src/app/modules/item/pages/item-by-id/item-by-id.component.ts

```diff
@@ -55,7 +55,8 @@ export class ItemByIdComponent implements OnDestroy {
 
     // on route change or user change: refetch item if needed
     this.activatedRoute.paramMap.pipe(
-      repeatLatestWhen(this.userSessionService.userChanged$)
+      repeatLatestWhen(this.userSessionService.userChanged$),
+      filter(() => !(history.state as Record<string, boolean>).preventRefetch),
```

Comment:
a quick comment may be useful to remember what it is for

Reply:
fixed